### ISSUE TITLE
TableView insets not adjusted properly for table with height constraint

### ISSUE THREAD

**Issue by puzlpix** [uias/Tabman#236]
### New Issue Checklist

- [x] Updated Tabman to the latest version
- [x] I took a look at the [Troubleshooting Guide](https://github.com/uias/Tabman/blob/master/Docs/TROUBLESHOOTING.md)
- [x] I searched for [existing GitHub issues](https://github.com/uias/Tabman/issues)

### Issue Description

Swift 4, iOS 10.3

When pinning all tableView edges to superview edges, the tableView top inset is adjusted as expected.

```
tableView.autoPinEdgesToSuperviewEdges()
```

But when pinning top, leading, and trailing to superview edges, but not pinning bottom and instead setting height dimension, tableView top inset is about 88pt down from where it should be.

```
tableView.autoPinEdge(toSuperviewEdge: .top)
tableView.autoPinEdge(toSuperviewEdge: .leading)
tableView.autoPinEdge(toSuperviewEdge: .trailing)
tableView.autoSetDimension(.height, toSize: Constants.CellDefaultHeight)
```
TableView height is fixed to leave room on screen for a second scrollView where its top is pinned to bottom of tableView.

**Comment by msaps** [comment]
@puzlpix okay sounds like something that should be fixed! Will check it out ASAP 😄

**Comment by msaps** [comment]
@puzlpix this should be resolved as of [1.5.0](https://github.com/uias/Tabman/releases/tag/1.5.0) 🍾
